app/assets/javascripts/discourse/tests/setup-tests.js

```diff
@@ -94,6 +94,8 @@ function createApplication(config, settings) {
       return container;
     });
 
+  SiteSettingService.create = () => settings;
```

Comment:
Hmm… don't like that ^ 👀 Irreversible code modification is a no-no 😛 I suppose we can replace it later with a proper registration.

Reply:
I know, I know, I hate it too 😭 

I spent many hours trying to `app.register("service:site-settings", settings, {instantiate: false})` working per [the docs](https://guides.emberjs.com/release/testing/testing-components/#toc_stubbing-services). But, whatever I did, the definition in `discourse/services/site-settings.js` would always 'win' in a lookup. Not sure if that's an Ember issue, or an issue with our slightly weird test environment.

Reply:
I'm almost certain that's because of our test setup **and** our app initialization logic.

Reply:
If we can fix it that would be awesome, but I'm tempted to merge this as-is for now. I know it's not ideal, but it's clearly an improvement over what we had before.

Reply:
Yup, that's why I already approved this PR 😁